modules/oe_theme_inpage_navigation/src/Plugin/EntityMetaRelation/InPageNavigation.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_inpage_navigation\Plugin\EntityMetaRelation;
+
+use Drupal\Core\DependencyInjection\DependencySerializationTrait;
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\emr\Entity\EntityMetaInterface;
+use Drupal\emr\Plugin\EntityMetaRelationContentFormPluginBase;
+
+/**
+ * Provides an entity meta plugin to store if a node with inpage navigation.
+ *
+ * @EntityMetaRelation(
+ *   id = "inpage_navigation",
+ *   label = @Translation("Inpage Navigation"),
+ *   entity_meta_bundle = "oe_theme_inpage_navigation",
+ *   content_form = TRUE,
+ *   description = @Translation("Allows to configure a node with inpage navigation."),
+ *   entity_meta_wrapper_class = "\Drupal\oe_theme_inpage_navigation\InPageNavigationWrapper",
+ * )
+ */
+class InPageNavigation extends EntityMetaRelationContentFormPluginBase {
+
+  use StringTranslationTrait;
+  use DependencySerializationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build(array $form, FormStateInterface $form_state, ContentEntityInterface $entity): array {
+    $key = $this->getFormKey();
+    $this->buildFormContainer($form, $form_state, $key);
+
+    $entity_meta_bundle = $this->getPluginDefinition()['entity_meta_bundle'];
+
+    /** @var \Drupal\emr\Field\EntityMetaItemListInterface $entity_meta_list */
+    $entity_meta_list = $entity->get('emr_entity_metas');
+    /** @var \Drupal\emr\Entity\EntityMetaInterface $entity_meta */
+    $entity_meta = $entity_meta_list->getEntityMeta($entity_meta_bundle);
+    /** @var \Drupal\oe_theme_inpage_navigation\InPageNavigationWrapper $entity_meta_wrapper */
+    $entity_meta_wrapper = $entity_meta->getWrapper();
+
+    $form[$key]['inpage_navigation'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Enable Inpage navigation'),
```

Comment:
The form elements should use the values from the requirements page: https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?pageId=524912512 (e.g. "Activate in Page Menu").